CHANGES.rst

```diff
@@ -14,6 +14,52 @@ Changelog
 
 .. towncrier release notes start
 
+6.5.1
+=====
+
+*(2025-06-24)*
+
+
+Bug fixes
+---------
+
+- Fixed a bug in C implementation when during multidict resizing the function could drop
+  last item under some circumstances.
+
+  The bug was introduced by multidict 6.5.0 release.
+
+  Patch by :user:`asvetlov`.
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`1195`.
+
+
+Contributor-facing changes
+--------------------------
+
+- Setup pre-commit hook for auto-formatting both Python and C code.
```

Comment:
```suggestion
- A pair of code formatters for Python and C have been configured in the pre-commit tool.
```

Reply:
As you wish. 
Technically 'setup' verb has the same spelling in both present and past forms, isn't it?

Reply:
No, it's not. `set up` is a verb or an adjective. `setup` or `set-up` is a noun.